sanicbot/extensions/help.py

```diff
@@ -0,0 +1,13 @@
+from discord.ext import commands
+
+
+class Help(commands.Cog):
```

Comment:
By switching to application (slash) commands, this cog becomes obsolete. A user simply needs to start typing `/` to get a list of commands and their descriptions.